rules/manager.go

```diff
@@ -21,6 +21,8 @@ import (
 	"sync"
 	"time"
 
+	"golang.org/x/net/context"
+
```

Comment:
Yeah, that one should have gone into the existing external package import block below, will fix.